parsers/nmparser/parse_lst_file.go

```diff
@@ -107,11 +291,31 @@ func ParseLstEstimationFile(lines []string) ModelOutput {
 			shrinkageDetails = parseShrinkage(line, shrinkageDetails)
 		case strings.Contains(line, "EPSSHRINK"):
 			shrinkageDetails = parseShrinkage(line, shrinkageDetails)
+		case strings.Contains(line, "0MINIMIZATION SUCCESSFUL"):
+			runHeuristics.MinimizationSuccessful = HeuristicTrue
+		case strings.Contains(line, "GRADIENT:"):
+			gradientLines = append(gradientLines, line)
+		case strings.Contains(line, "RESET HESSIAN"):
+			runHeuristics.HessianReset = HeuristicTrue
+		case strings.Contains(line, "PARAMETER ESTIMATE IS NEAR ITS BOUNDARY"):
+			runHeuristics.ParameterNearBoundary = HeuristicTrue
+		case strings.Contains(line, "COVARIANCE STEP OMITTED: NO"):
+			runHeuristics.CovarianceStepOmitted = HeuristicTrue
+		case strings.Contains(line, "EIGENVALUES OF COR MATRIX OF ESTIMATE"):
+			// TODO: get largeNumberLimit from config
+			// or derive. something like (number of parameters) * 10
+			runHeuristics.LargeConditionNumber = getLargeConditionNumberStatus(lines, i, 1000.0)
```

Comment:
how did you choose a value of 1000?

Reply:
PsN default is 1000.

https://github.com/UUPharmacometrics/PsN/blob/88320646310b63f0c88075a6a71f4d5a87be4528/bin/sumo#L60

From sumo doc:
-condition_number_limit = N
Default is 1000. The cut-off for issuing a warning for a high condition number. The test will only be made if the $COV step has
been requested in the NMtran control stream and NONMEM
has been asked to report eigenvalues ($COV PRINT=E).